XamarinCommunityToolkit.UnitTests/Xamarin.CommunityToolkit.UnitTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net461</TargetFrameworks>
```

Comment:
since netcoreapp3.1 is added for wpf, UTs for snackbar don't work. added net461 for Snackbar tests